# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1531772939 14400
#      Mon Jul 16 16:28:59 2018 -0400
# Node ID 6ff900fd15cb44cb11d8d50645b29453b90b2169
# Parent  f89bf246dcd4639fdd8b7edfdccfbbe5b9f2b64e
avoid shadowed variable warning

* file-editor.cc (file_editor::request_open_file):
Eliminate duplicate settings variable.

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -1301,17 +1301,16 @@ namespace octave
                         msgBox->setAttribute (Qt::WA_DeleteOnClose);
                         msgBox->show ();
                       }
                     else
                       {
                         // File does not exist, should it be created?
                         bool create_file = true;
                         QMessageBox *msgBox;
-                        QSettings *settings = resource_manager::get_settings ();
 
                         if (! settings->value ("editor/create_new_file", false).toBool ())
                           {
                             msgBox = new QMessageBox (QMessageBox::Question,
                                                       tr ("Octave Editor"),
                                                       tr ("File\n%1\ndoes not exist. "
                                                           "Do you want to create it?").arg (openFileName),
                                                       QMessageBox::NoButton,nullptr);
